gravitee-apim-portal-webui/src/app/pages/application/application-general/application-general.component.ts

```diff
@@ -144,7 +143,7 @@ export class ApplicationGeneralComponent implements OnInit, OnDestroy {
             grant_types: new FormArray([]),
           }),
           tls: this.formBuilder.group({
-            client_certificate: new FormControl(this.application.settings.tls.client_certificate, null),
+            client_certificate: new FormControl(this.application.settings?.tls?.client_certificate, null),
```

Comment:
Haha,  I've just posted a message on a previous commit for this. 
like this is 👌   but this commit can be fixup the one before :p